### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bzantium**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 </div>
 
 ## What is LASSL
-LASSL is a **LA**nguage library for **S**elf-**S**upervised **L**earning. LASSL aims to provide a easy-to-use framework for pretraining language model by only using Huggingface's Transformers and Datasets.
+LASSL is a **LA**nguage library for **S**elf-**S**upervised **L**earning. LASSL aims to provide an easy-to-use framework for pretraining language model by only using Huggingface's Transformers and Datasets.
```
Comment (seopbo) [inline README.md]:
```suggestion
LASSL is a **LA**nguage framework for **S**elf-**S**upervised **L**earning. LASSL aims to provide an easy-to-use framework for pretraining language model by only using Huggingface's Transformers and Datasets.
```

**Review by iron-ij — DISMISSED** [review]
💯

File: README.md (hunk 1 above)
Comment (seopbo) [reply README.md]: I think that we made a framework not library. Can we change this? @bzantium 

ref: https://webclub.tistory.com/458

**Review by seopbo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (iron-ij) [reply README.md]: Thanks for good reference, I learned the difference of Framework, Library

**Review by seopbo — APPROVED** [review]
LGTM

**Review by iron-ij — APPROVED** [review]
LGTM 👍

**Pull request merged**